packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -30,7 +30,7 @@ export const styles = (theme: ThemeType): JssStyles => ({
   },
   imageContainer: {
     width: '100%',
-    '& > img': {
+    '& > picture > img': {
```

Comment:
Hmm I'm a bit concerned about the image -> picture change breaking other places. Have you checked the other places CloudinaryImage2 is used?

Reply:
I have not checked all uses of CloudinaryImage2, but I've checked for '> img', and the ones I expect to be dicey.